src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -49,7 +51,17 @@ public record ConnectionSpec(
                           + "Apache Kafka cluster.")
     @JsonProperty(KAFKA_CLUSTER_CONFIG_FIELD_NAME) KafkaClusterConfig kafkaClusterConfig,
     @Schema(description = "The details for connecting to a Schema Registry.")
-    @JsonProperty(SCHEMA_REGISTRY_CONFIG_FIELD_NAME) SchemaRegistryConfig schemaRegistryConfig
+    @JsonProperty(SCHEMA_REGISTRY_CONFIG_FIELD_NAME) SchemaRegistryConfig schemaRegistryConfig,
```

Comment:
Ooh, this pattern established above is harder to read. Maybe we should be consistent and have all annotations be on lines before the actual record parameter type and name.
```suggestion
    @JsonProperty(SCHEMA_REGISTRY_CONFIG_FIELD_NAME)
    SchemaRegistryConfig schemaRegistryConfig,
```